### DISCUSSION TITLE
Help page

### DISCUSSION THREAD

**Discussion by gcelano** [korpling/ANNIS#875] [General]
the Help page can be changed using "`help-url": "https://example.com/annis-help"` in `instances/xxxx.json`, but where can the template be found?

**Comment by thomaskrause** [comment]
We use [mdBook](https://rust-lang.github.io/mdBook/) to create the documentation. The markdown files for the included ANNIS help are located in the git repo: <https://github.com/korpling/ANNIS/tree/main/docs/online-help/>. The [`SUMMARY.md`](https://github.com/korpling/ANNIS/blob/main/docs/online-help/src/SUMMARY.md) file in the `src` folder defines the structure and the pages. Most of these pages are actually links to the user guide, e.g. the page <https://github.com/korpling/ANNIS/blob/main/docs/online-help/src/interface/result-window.md> has the special  `{{#include ../../../user-guide/src/interface/result-window.md}}` command which includes the content of the linked markdown file. In this specific case the actual content is thus located in <https://github.com/korpling/ANNIS/blob/main/docs/user-guide/src/interface/result-window.md>
